demos/build_demos_msvc.bat

```diff
@@ -34,6 +34,14 @@ if not "%1" == "" (
         shift & shift
         goto argParse
     )
+    rem to build more than one specific demo use quotation marks,
+    rem list the necessary demos comma-separated and without spaces,
+    rem ex. -DDEMOS="classification_demo,segmentation_demo"
+    if "%1" == "-DDEMOS" (
+        set EXTRA_CMAKE_OPTS=%EXTRA_CMAKE_OPTS% %1=%2
+        shift & shift
+        goto argParse
+    )
```

Comment:
CMake creates a target per each demo. You should run cmake like `cmake --build . --config Release --target object_detection_demo_ssd_async segmentation_demo` instead of avoiding adding subdirectory.

Reply:
Fixed